task_10/andrew.samsonov/src/test/java/ua/pp/darknsoft/StringOpportunityTest.java

```diff
@@ -0,0 +1,28 @@
+package ua.pp.darknsoft;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+public class StringOpportunityTest {
+
+  @Test
+  public void replacementOfSymbolsTest() {
+    String testStr = StringsOpportunity.replacementOfSymbols("teacherluke@podomatic.com");
+    assertThat(testStr, is("teacherluke[ at ]podomatic[ dot ]com"));
+  }
+
+  @Test
+  public void numericValueOfSymbolsTest() {
```

Comment:
Could you also add a test for **cat** as Sergey suggested.
catShouldReturnCat, i.e. the same word